docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -307,9 +307,13 @@ You can make further modifications to a saved filter view later using kebab menu
 
 ## Create an SLO from Metrics page
 
-:::note Coming soon
-You'll be able to create SLOs from Metrics Explorer.
-:::
+To create an SLO from the **Metrics** page:
+
+1. Click xyz...
```

Comment:
You can create SLOs with operators, provided you use an appropriate quantization.

```suggestion
1. Go to **Manage Data** > **Metrics**.
1. Under **Metrics Explorer**, select your desired **Metric** and **Filters**. Optionally, you can **Add Operator**.
1. Click the three-dot kebab icon, then select **Create SLO**.
1. Follow the instructions under [Create an SLO (General)](#create-an-slo-general).
```